Update CONTRIBUTING.md (#340)
openvinotoolkit/openvino_notebooks: CONTRIBUTING.md

1. Always provide links to sources. If your notebook implements a model, link to the research paper
   and the source Github (if available).

> 2. Only use data and models that have a license that permits usage for commercial purposes, and make sure to

Reviewer: ```suggestion
2. Use only data and models with permissive licenses that allow for commercial use, and make sure to
```
This made more sense to me but open to other suggestions
